pkg/token/token.go

```diff
@@ -570,7 +578,11 @@ func (v tokenVerifier) Verify(token string) (*Identity, error) {
 
 	metrics.Get().StsResponses.WithLabelValues(fmt.Sprint(response.StatusCode)).Inc()
 	if response.StatusCode != 200 {
-		return nil, NewSTSError(fmt.Sprintf("error from AWS (expected 200, got %d). Body: %s", response.StatusCode, string(responseBody[:])))
+		responseStr := string(responseBody[:])
+		if strings.Contains(responseStr, "Throttling") {
```

Comment:
I assume you are referencing https://docs.aws.amazon.com/STS/latest/APIReference/CommonErrors.html, can you add a link to that in a comment?

Reply:
that's a good point. thank you! I will add a link to it in comment